### PR TITLE
Fix for video sharing

### PR DESCRIPTION
**Pull request by sebworks**
Fix for video sharing

## Removals

- Removed `cfgov/jinja2/v1/_includes/macros/share.html` because it's no longer needed.

## Changes

- Modified `cfgov/jinja2/v1/_includes/macros/video-player.html` to use `social-media` molecule.  Added static properties necessary to share content via the `social-media` molecule. 

- Modified `cfgov/unprocessed/css/video-player.less` to reference `social-media` molecule. 

## Testing
- run `gulp styes`
- Visit `http://localhost:8000/about-us/the-bureau/` and use the share buttons. The share buttons should work as expected ( Facebook share button won't work on localhost ). 

## Screenshots
- Email
screen shot 2017-01-23 at 4 27 13 pm

-  Twitter
screen shot 2017-01-23 at 4 27 57 pm

- LinkedIn
screen shot 2017-01-23 at 4 28 33 pm

- Buttons
screen shot 2017-01-23 at 4 30 20 pm

## Notes

- The old text states `SHARE THIS VIDEO` and the new text states `SHARE THIS`. 
- Added the LinkedIn share button.
- At some point we will need to convert the video-player into a atomic component.

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

**Comment by Scotchester** [comment]
This is a good start, @sebworks. It's not quite working in the FCM layout of the video player, though:

screen shot 2017-01-26 at 17 38 38

File: cfgov/unprocessed/css/video-player.less [hunk 1]
```diff
@@ -162,59 +162,68 @@
...
+    .m-social-media_icon {
         color: @pacific;
     }
 
-    .respond-to-min(@bp-sm-min, {
-        text-align: right;
-    });
+    .m-social-media_icons {
+    	display: block;
+    	margin-right: unit( 5px / @base-font-size-px, em);
```
Comment (Scotchester) [inline cfgov/unprocessed/css/video-player.less]: Trivial: Please use 4-space indents here. The two lines above each have four spaces and one tab. Why the tab is rendering as 2-characters wide, I have no idea.

File: cfgov/unprocessed/css/video-player.less [hunk 2]
```diff
@@ -162,59 +162,68 @@
...
         left: auto;
     }
-});
+} );
+
+.respond-to-range( @bp-xs-max, @fcm-bp-min, {
+	.video-player .m-social-media {
+		text-align: right;
+	}
+} );
```
Comment (Scotchester) [inline cfgov/unprocessed/css/video-player.less]: Trivial: Please use 4-space indents here. This block's using tabs, which are rendering in the classic **8**-character width of the GitHub diff view.

**Comment by Scotchester** [comment]
It's probably worth pointing out here that the buttons sitting atop the video in an FCM was _already_ happening, if that FCM was on a page with a right sidebar (which has a narrower main content column than pages with a left sidebar):



Please try to fix it so that they don't overlap in these smaller-width FCMs, too.

**Comment by Scotchester** [comment]
I should say that the crux of the PR—fixing the operation of the share buttons on the video player—is working for me locally :)

**Comment by sebworks** [comment]
@Scotchester, did you run gulp styles? I'm not sure how you got that.

**Comment by Scotchester** [comment]
Hah, no I did not. Thanks for the reminder. I'll check it out in the
morning.

**Comment by sebworks** [comment]
That is my fault. I didn't add it to the test notes

**Comment by Scotchester** [comment]
Ok, looking good on a wider FCM like about/the-bureau. Still overlapping on a narrower one like /prepaid-rule, but since that wasn't technically the scope of this PR, we can address it in another one.

**Pull request merged**